tests/slurm_plugin/test_resume.py

```diff
@@ -167,23 +167,8 @@ def test_resume_config(config_file, expected_attributes, test_datadir, mocker):
                 client_error("RequestLimitExceeded"),
             ],
             {"RequestLimitExceeded": {"queue1-st-c5xlarge-2"}},
-            [
-                call(
-                    ["queue1-dy-c5xlarge-1", "queue1-dy-c5xlarge-2", "queue1-st-c5xlarge-1"],
-                    nodeaddrs=["ip.1.0.0.1", "ip.1.0.0.2", "ip.1.0.0.3"],
-                    nodehostnames=None,
-                )
-            ],
-            dict(
-                zip(
-                    ["queue1-dy-c5xlarge-1", "queue1-dy-c5xlarge-2", "queue1-st-c5xlarge-1"],
-                    [
-                        EC2Instance("i-11111", "ip.1.0.0.1", "ip-1-0-0-1", datetime(2020, 1, 1, tzinfo=timezone.utc)),
-                        EC2Instance("i-22222", "ip.1.0.0.2", "ip-1-0-0-2", datetime(2020, 1, 1, tzinfo=timezone.utc)),
-                        EC2Instance("i-33333", "ip.1.0.0.3", "ip-1-0-0-3", datetime(2020, 1, 1, tzinfo=timezone.utc)),
-                    ],
-                )
-            ),
+            [],
```

Comment:
Why these parameters are empty now?

Reply:
Answer: these parameters are empty because now there is a real ALL_OR_NOTHING strategy where we're assigning nodes only if all of them are available.  Previously it was "wrongly" using a "best-effort" strategy.